### ISSUE TITLE
Game Bug: Chibi-Robo audio issue

### ISSUE THREAD

**Issue by cheatfreak47** [FIX94/Nintendont#485]
Occasionally audio randomly cuts out or loops wrong, creating some really jarring silence in the middle of the game. Happens frequently during the game's opening cutscene where the music is _supposed_ to start and stop, but immediately cuts out often seconds after it starts instead of continuing as intended.

Seems to happen on all version of the game as well. 
Apparently this can also happen on dolphin, but not on real hardware, so something must need to be fixed to correct the problem.

**Comment by GB-CG** [comment]
Not a bug on my end. I use 5.466 with the verified US version. Also mentioning Dolphin is kinda pointless for that.

**Comment by cheatfreak47** [comment]
@GB-CG I have the same ISO and I'm using the same version, and the issue is happening consistently on my end. 🤔

I'll try resetting my configuration and formatting my external devices, maybe the problem is with my storage devices?

**Comment by cheatfreak47** [comment]
@GB-CG *facepalm* Looks like it was a user-issue, semi-related to a problem I've been having with my SD card and saving. I think I just need a new SD card.

The problem was "unlock read speed limit" was turned on even after I went and changed the setting because it seems like Nintendont doesn't like saving on my SD card, Once I dropped a new config file on the SD with the setting changed, the audio issue disappeared so this game must be sensitive to disc read timings.
